.github/workflows/builds.yml

```diff
@@ -221,6 +221,9 @@ jobs:
                 sudo \
                 which \
                 zlib-devel
+          # Packages needed for some diagrams generation.
+          dnf -y install epel-release
+          dnf -y install mscgen
           # These packages required to build the PDF.
           yum -y install \
```

Comment:
Q: why do we use `yum` for all the packages?

Reply:
As of CentOS 8 (Alma, RHEL, etc) `yum` is just an alias to `dnf`.  When I wrote this, I probably still had the muscle memory from CentOS 7 days where there was only `yum`.

Reply:
oooh didn't know it was an alias now. Thank you!